File: src/main/java/tn/esprit/spring/model/ChangePasswordRequest.java
package tn.esprit.spring.model;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;
@Builder
@Getter
@Setter
public class ChangePasswordRequest {

    private String oldPass;
    private String newPass;
    private String email;

    public String getOldPass() {
        return oldPass != null ? oldPass : "";
    }

}

File: src/app/components/models/ChangePasswordRequest.ts
export class ChangePasswordRequest {
    oldPass: string;
    newPass: string;
    email: string;
  
    constructor(oldPass: string, newPass: string, email: string) {
      this.oldPass = oldPass;
      this.newPass = newPass;
      this.email = email;
    }
  }